### ISSUE TITLE
Could you release version 1.2.0?

### ISSUE THREAD

**Issue by minhna** [johanbrook/meteor-publication-collector#44]
Or any version which support `async` publishing?

**Comment by derwaldgeist** [comment]
I need this for my Meteor 3 update, too. I am wondering why 1.2.0 is not on Packosphere yet? It seems to be finished.

**Comment by derwaldgeist** [comment]
I tried to add it manually in the `packages` folder, but it doesn't work because it depends on `mongo@1.1.5`.

The current master 2.0.0-rc.0 seems to work, though.

**Comment by harry-73** [comment]
Hi,

Is it excepted to publish release 2.0.0 ?

Thanks

**Comment by derwaldgeist** [comment]
I forked it and added it as a local package.

**Comment by harry-73** [comment]
> I forked it and added it as a local package.

 Yes, I did but it would better to have it published to avoid local package ;-)

**Comment by nachocodoner** [comment]
This package will likely be moved to core in Meteor 3.4 or shortly after.

It is important for testing publications. It’s used by skel-full, which [we’re fixing](https://github.com/meteor/skel-full/pull/1#pullrequestreview-3067190598). Given its reliance on core testing, it’s a good fit for core. It’s simple and self-contained.

Also consider @alisnic’s fixes [here](https://github.com/Meteor-Community-Packages/meteor-publication-collector/pull/43) when moving it to core.

**Comment by jankapunkt** [comment]
@nachocodoner there were a few issues found in the newest code, which I fixed and published an intermediate versions under `communitypackages:publication-collector@2.0.0-rc.1`

**Comment by jankapunkt** [comment]
Please also see https://github.com/Meteor-Community-Packages/meteor-publication-collector/pull/45
